server/libs/domain/src/system-config/system-config.core.ts

```diff
@@ -11,9 +11,10 @@ const defaults: SystemConfig = Object.freeze({
   ffmpeg: {
     crf: '23',
     preset: 'ultrafast',
-    targetVideoCodec: 'libx264',
+    targetVideoCodec: 'h264',
```

Comment:
This option causes quicktime video doesn't have sound anymore

Reply:
It'll be the MP3 audio, I will make a change to switch that to AAC then you can retest with that